packages/@aws-cdk/aws-scheduler-alpha/lib/input.ts

```diff
@@ -0,0 +1,120 @@
+import { DefaultTokenResolver, IResolveContext, Stack, StringConcat, Token, Tokenization } from 'aws-cdk-lib';
+import { ISchedule } from './schedule';
+
+/**
+ * The text, or well-formed JSON, passed to the target of the schedule.
+ */
+export abstract class ScheduleTargetInput {
+  /**
+   * Pass text to the target, it is possible to embed `ContextAttributes` and other
+   * cdk references.
+   *
+   * The target input value will be a single string: the string you pass
+   * here. Do not use this method to pass a complex value like a JSON object to
+   * a target. Use `ScheduleTargetInput.fromObject()` instead.
+   * 
+   * @param text text to use as input for the target
```

Comment:
Should we separate visually param name and description? Could also add `-`?
```suggestion
   * @param text Text to use as the input for the target
```